_metadata/repos.conf.d/ubuntu/provider.conf

```diff
@@ -1,18 +1,11 @@
 dists/*-proposed/
 dists/*-backports/
-dists/*/universe/
-dists/*/multiverse/
-*i386/
 dists/devel/
 dists/devel-*/
 dists/precise-*/
```

Comment:
Precise is still being mirrored in upstream, so we should do so as well. Yakkety and Zesty are non-LTS releases and have been dropped upstream, so we can pull these filters out. Agree with removing universe and multiverse filters.